src/index.js

```diff
@@ -1,14 +1,40 @@
-import _ from "lodash";
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const list = document.querySelector('.toDoList');
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const toDoList = [
+  {
+    description: 'task 1',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'task 2',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'task 3',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'task 4',
+    completed: true,
+    index: 4,
+  },
+];
 
-  return element;
+let $toDoList = '';
+
+for (let i = 0; i < toDoList.length; i += 1) {
+  $toDoList += `
+    <li>
+      <input type="checkbox">
+      <p>${toDoList.description}</p>
+      <p>${toDoList.completed}</p>
+    </li>
+    `;
 }
 
-document.body.appendChild(component());
\ No newline at end of file
+list.innerHTML = $toDoList;
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Write%20a%20function%20to%20iterate%20over%20the%20tasks%20array%20and%20populate%20an%20HTML%20list%20item%20element%20for%20each%20task.), Please include all the code responsible for iterating over the tasks array and populating an HTML list item element for each task inside a function.

- [ ] the data you're rendering on the DOM is undefined, Kindly fix this error.

![image](https://github.com/sahlminkok/To-Do-List/assets/68796232/7206de8a-06c5-49bb-b99f-36a3d869deae)